realsense_camera/README.md

```diff
@@ -88,13 +88,13 @@ IR2 camera: Available only for <b>R200</b> cameras.
         Calibration data
 
 ####Services
-    get_settings (camera/get_settings)
+    get_settings (camera/driver/get_settings)
```

Comment:
@kevincwells is this a **standard** to have the "driver" part of the service names, like "camera/driver/get_settings" instead of just "camera/get_settings"? or has this been done because the **code change** automatically appends the nodelet name to the service name?